### PR TITLE
Themes: Add feature list to theme sheets

### PR DESCRIPTION
**Pull request by seear**
Addresses #3163

Work in progess.

@mendezcode: This would be a good task for you to continue.

### CODE REVIEW

File: client/my-sites/themes/sheet.jsx [hunk 1]
```diff
@@ -141,6 +148,12 @@ const ThemeSheet = React.createClass( {
 						<div className="themes__sheet-content">
 							{ this.renderSectionNav( section ) }
 							<Card className="themes__sheet-content">{ themeContentElement }</Card>
+							<SectionHeader label={ i18n.translate( 'Features' ) } />
```
Comment (seear) [inline client/my-sites/themes/sheet.jsx]: Might be good to extract this whole part (plus the logic from above) into a `renderFeaters()` func like e.g. `renderBar()` to keep things tidy.

File: client/my-sites/themes/style.scss [hunk 2]
```diff
@@ -196,6 +196,32 @@
...
+		color: $gray-dark;
+		font-size: 0.925em;
+		margin: 0.38em 0.37rem;
+		padding: 0.25em 0.7em;
+		border-radius: 3px;
+
+		span {
+			display: block;
+
+			&:hover {
```
Comment (seear) [inline client/my-sites/themes/style.scss]: This part not necessary. Would only have been necessary if we were using `<a>` tags already. My fault for confusing you :)

**Comment by seear** [comment]
Looking really good to me. /cc @folletto for design review.

@mendezcode: see if you can fix up the test that I broke with my commit before you touched anything.
(`client/state/themes/theme-details/test/reducer.js`)

**Comment by folletto** [comment]
Task tested:

1. Open My Sites → Switch to All Sites.
2. Open Themes, pick any theme and select ••• → Details
3. Check the theme "Features" card down the page. 
4. Verify that the list matches the one on the same theme on theme.wordpress.com

Tested on Chrome, Firefox, Safari. 
Looks good. :+1: 
 
I just noticed that we're showing more tags than the old v2.0 showcase:

screen shot 2016-03-18 at 17 10 28

It's fine for me, I'm mostly curious why some are filtered out in the old showcase.

**Comment by seear** [comment]
> I'm mostly curious why some are filtered out in the old showcase.

I don't know, but showcase 4.0 shows the full list

screen shot 2016-03-21 at 16 23 46

File: client/my-sites/themes/sheet.jsx [hunk 3]
```diff
@@ -114,6 +116,24 @@ const ThemeSheet = React.createClass( {
...
+		const themeFeatures = this.props.taxonomies && this.props.taxonomies.features instanceof Array
+		? this.props.taxonomies.features.map( function( item, i ) {
+			return ( <li key={ 'theme-features-item-' + i++ }><span>{ item.name }</span></li> );
+		} ) : [];
+
+		return (
+			<div>
+				<SectionHeader label={ i18n.translate( 'Features' ) } />
+				<Card className="themes__sheet-features">
+					<ul className="theme-features-list">
```
Comment (seear) [inline client/my-sites/themes/sheet.jsx]: `theme-features-list` doesn't quite conform to the css class naming guidelines:
https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md

(Also may not need the `themes__sheet-features` class at all)

**Comment by seear** [comment]
After the css class naming tweaks this will be ready to merge :+1:

**Comment by folletto** [comment]
:boom:

**Comment by mendezcode** [comment]
:sweat_smile:

**Pull request merged**